hexa/databases/graphql/schema.graphql

```diff
@@ -0,0 +1,30 @@
+type Database {
+  tables(page:Int, perPage:Int): DatabaseTablePage!
+}
+
+type TableColumn {
+  name: String!
+  type: String!
+  sample: [[String]]
+}
+
+
+type DatabaseTable{
+  name: String!
+  totalCount: Int!
+  columns: [TableColumn!]!
+  sample:[[String]]
+  database: Database!
```

Comment:
When we try to resolve a databaseTableColumn, we need to have access to the db url in order to open a connection. I didn't have access to the database object in the context so that's why I added it here.

Reply:
Yes I remember but I think that if you implement it like I commented at the end of the review and if you return the workspace for the Database object you will be able to have it

Reply:
Okay i'll update it @qgerome